{
  message: 'Database not connected',
  error: Error: queryTxt ETIMEOUT cluster0.n72f5gi.mongodb.net
      at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:251:17) {
    errno: undefined,
    code: 'ETIMEOUT',
    syscall: 'queryTxt',
    hostname: 'cluster0.n72f5gi.mongodb.net'
  },
  level: 'error',
  service: 'user-service',
  label: 'PH',
  timestamp: '2023-07-01T07:02:34.001Z'
}
{
  message: 'Database not connected',
  error: Error: queryTxt ETIMEOUT cluster0.n72f5gi.mongodb.net
      at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:251:17) {
    errno: undefined,
    code: 'ETIMEOUT',
    syscall: 'queryTxt',
    hostname: 'cluster0.n72f5gi.mongodb.net'
  },
  level: 'error',
  service: 'user-service',
  label: 'PH',
  timestamp: '2023-07-01T07:05:09.753Z'
}
{
  message: 'Database not connected',
  error: MongooseServerSelectionError: connect ECONNREFUSED ::1:27017
      at _handleConnectionErrors (G:\Mission-3 (Project)\UM-App\node_modules\mongoose\lib\connection.js:792:11)
      at NativeConnection.openUri (G:\Mission-3 (Project)\UM-App\node_modules\mongoose\lib\connection.js:767:11) {
    reason: TopologyDescription {
      type: 'Unknown',
      servers: Map(1) {
        'localhost:27017' => ServerDescription {
          address: 'localhost:27017',
          type: 'Unknown',
          hosts: [],
          passives: [],
          arbiters: [],
          tags: {},
          minWireVersion: 0,
          maxWireVersion: 0,
          roundTripTime: -1,
          lastUpdateTime: 650208841,
          lastWriteDate: 0,
          error: MongoNetworkError: connect ECONNREFUSED ::1:27017
              at connectionFailureError (G:\Mission-3 (Project)\UM-App\node_modules\mongodb\src\cmap\connect.ts:491:14)
              at Socket.<anonymous> (G:\Mission-3 (Project)\UM-App\node_modules\mongodb\src\cmap\connect.ts:392:16)
              at Object.onceWrapper (node:events:628:26)
              at Socket.emit (node:events:513:28)
              at emitErrorNT (node:internal/streams/destroy:151:8)
              at emitErrorCloseNT (node:internal/streams/destroy:116:3)
              at processTicksAndRejections (node:internal/process/task_queues:82:21) {
            cause: Error: connect ECONNREFUSED ::1:27017
                at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1300:16) {
              errno: -4078,
              code: 'ECONNREFUSED',
              syscall: 'connect',
              address: '::1',
              port: 27017
            },
            [Symbol(errorLabels)]: Set(1) { 'ResetPool' }
          },
          topologyVersion: null,
          setName: null,
          setVersion: null,
          electionId: null,
          logicalSessionTimeoutMinutes: null,
          primary: null,
          me: null,
          '$clusterTime': null
        }
      },
      stale: false,
      compatible: true,
      heartbeatFrequencyMS: 10000,
      localThresholdMS: 15,
      setName: null,
      maxElectionId: null,
      maxSetVersion: null,
      commonWireVersion: 0,
      logicalSessionTimeoutMinutes: null
    },
    code: undefined
  },
  level: 'error',
  service: 'user-service',
  label: 'PH',
  timestamp: '2023-07-01T07:06:33.956Z'
}
